ncm-metaconfig/src/main/metaconfig/ganesha/1.5/tests/regexps/config/gpfs

```diff
@@ -0,0 +1,5 @@
+Tests for gpfs config
+---
+/etc/ganesha/ganesha.nfsd.conf
+---
+^GPFS {
```

Comment:
already existing test in master, git did not recognise the move. Anyway, in 1.5 GPFS block did not contain anything